tests/controllers/test_mock_party_controller.py

```diff
@@ -0,0 +1,161 @@
+import unittest
+
+from response_operations_ui.controllers import mock_party_controller
+
+example_data = [
+    {'firstName': 'Jillayne', 'lastName': 'Tran', 'emailAddress': 'jtran0@simplemachines.org'},
+    {'firstName': 'Virgie', 'lastName': 'Wiley', 'emailAddress': 'vwiley2@bing.com'},
+    {'firstName': 'Alikee', 'lastName': 'Banton', 'emailAddress': 'abanton1@dell.com'},
+    {'firstName': 'Tulley', 'lastName': 'Butts', 'emailAddress': 'tbutts3@google.pl'},
+    {'firstName': 'Mollie', 'lastName': 'Singyard', 'emailAddress': 'msingyard4@cargocollective.com'},
+    {'firstName': 'Mead', 'lastName': 'Wakefield', 'emailAddress': 'mwakefield5@ucoz.com'},
+    {'firstName': 'Hildegaard', 'lastName': 'Kubik', 'emailAddress': 'hkubik6@cmu.edu'},
+    {'firstName': 'Ami', 'lastName': 'Brimble', 'emailAddress': 'abrimble7@nasa.gov'},
+    {'firstName': 'Sidonia', 'lastName': 'Piddle', 'emailAddress': 'spiddle8@sakura.ne.jp'},
+    {'firstName': 'Raquela', 'lastName': 'Grzelewski', 'emailAddress': 'rgrzelewski9@mlb.com'},
+    {'firstName': 'Gerry', 'lastName': 'Tidbold', 'emailAddress': 'gtidbolda@bluehost.com'},
+    {'firstName': 'Cammy', 'lastName': 'Childerley', 'emailAddress': 'cchilderleyb@google.es'},
+    {'firstName': 'Sutherland', 'lastName': 'Barrett', 'emailAddress': 'sbarrettc@symantec.com'},
+    {'firstName': 'August', 'lastName': 'Moss', 'emailAddress': 'amossd@hubpages.com'},
+    {'firstName': 'Gualterio', 'lastName': 'Armsden', 'emailAddress': 'garmsdene@nifty.com'},
+    {'firstName': 'Garnette', 'lastName': 'Harrie', 'emailAddress': 'gharrief@netvibes.com'},
+    {'firstName': 'Zelda', 'lastName': 'Hale', 'emailAddress': 'zhaleg@paginegialle.it'},
+    {'firstName': 'Kathryn', 'lastName': 'Goose', 'emailAddress': 'kgooseh@tmall.com'},
+    {'firstName': 'Daisy', 'lastName': 'Roddan', 'emailAddress': 'droddani@smh.com.au'},
+    {'firstName': 'Magda', 'lastName': 'Hussey', 'emailAddress': 'mhusseyj@imgur.com'},
+    {'firstName': 'Fan', 'lastName': 'Wiggam', 'emailAddress': 'fwiggamk@theatlantic.com'},
+    {'firstName': 'Hadleigh', 'lastName': 'Duffyn', 'emailAddress': 'hduffynl@buzzfeed.com'},
+    {'firstName': 'Kelby', 'lastName': 'Dimblebee', 'emailAddress': 'kdimblebeem@ask.com'},
+    {'firstName': 'Torrey', 'lastName': 'Palin', 'emailAddress': 'tpalinn@technorati.com'},
+    {'firstName': 'Malinde', 'lastName': 'Featley', 'emailAddress': 'mfeatleyo@oakley.com'},
+    {'firstName': 'Chico', 'lastName': 'Thrasher', 'emailAddress': 'cthrasherp@cbc.ca'},
+    {'firstName': 'Thorvald', 'lastName': 'Puleston', 'emailAddress': 'tpulestonq@latimes.com'},
+    {'firstName': 'Hendrick', 'lastName': 'Pollins', 'emailAddress': 'hpollinsr@sfgate.com'},
+    {'firstName': 'Lyn', 'lastName': 'MacGaughie', 'emailAddress': 'lmacgaughies@webnode.com'},
+    {'firstName': 'Patrick', 'lastName': 'Mothersdale', 'emailAddress': 'pmothersdalet@istockphoto.com'}
+]
```

Comment:
is it worth sticking this in another file to reuse?

Reply:
At this stage, I don't think so, as it's only the data that this endpoint would return.  Wouldn't be difficult to if you're really keen on that happening though.

Reply:
Nah that's fair enough, wasn't sure if we'd have any use for a block of account details for anything else